src/app/common/util.service.js

```diff
@@ -208,7 +208,7 @@ module.exports.getPage = function (queryParams) {
  * @param defaultSort (optional)
  * @returns {Array}
  */
-module.exports.getSort = function (queryParams, defaultDir = 'ASC', defaultSort) {
+module.exports.getSort = function (queryParams, defaultDir = 'ASC', defaultSort = undefined) {
```

Comment:
SonarQube didn't like that there was a defaulted parameter before an un-defaulted parameter.